.travis.yml

```diff
@@ -178,13 +178,23 @@ jobs:
        apt-get -y install build-essential                                                 && \
        apt-get -y install git gfortran gcc g++ openmpi-bin libopenmpi-dev cmake           && \
        apt-get -y install libblas-dev liblapack-dev                                       && \
+       apt-get -y install libeigen3-dev                                                   && \
+       apt-get -y install python3-minimal python3-pip python3-numpy                       && \
+       pip3 install numpy                                                                 && \
+       apt-get -y install wget                                                            && \
+       wget https://sourceforge.net/projects/boost/files/boost/1.67.0/boost_1_67_0.tar.gz && \
+       tar -xf boost_1_67_0.tar.gz && cd boost_1_67_0                                     && \
+       ./bootstrap.sh --with-libraries=python --with-python=/usr/bin/python3              && \
+       ./b2 install                                                                       && \
+       apt-get install locate                                                             && \
+       updatedb                                                                           && \
        cd /tmp                                                                            && \
        cd arpack-ng                                                                       && \
        git status                                                                         && \
        git log -2                                                                         && \
        sed -e 's/mpirun /mpirun --allow-run-as-root --oversubscribe /' -i CMakeLists.txt  && \
        mkdir -p build && cd build                                                         && \
-       cmake -DEXAMPLES=ON -DMPI=ON -DICB=ON ..                                           && \
+       cmake -DEXAMPLES=ON -DMPI=ON -DPYTHON3=ON -DBOOST_PYTHON_LIBSUFFIX='36' ..         && \
```

Comment:
@sylvestre : as you want python3 only, BOOST_PYTHON_LIBSUFFIX _may_ change if ` apt-get upgrade ` change the default python version on bionic.

Reply:
I guess the suffix detection could be done automatically, no?

Reply:
Nope...

Reply:
Why?

Reply:
Don't know. If you don't set it, cmake is lost and find nothing. I think the reason is that all is done in such a way you can have a boost lib for any python version X.Y so automatic detection would not be so "obvious"...